src/index.ts

```diff
@@ -14,5 +18,38 @@ options.push({
   type: 'string',
   default: '',
 });
+options.push({
+  name: 'globalRepoURL',
+  description: 'URL of the global repo for the commodore manager',
+  type: 'string',
+  default: '',
+});
+options.push({
+  name: 'tenantId',
+  description:
+    'Tenant ID. This must match the tenant the repo is belonging to.',
+  type: 'string',
+  default: '',
+});
+
+// Patch renovate finalizer, we need the require() here despite what the TS
+// hint indicates. However, seems that suggestions cannot be suppressed with
+// typescript hints (e.g. @ts-ignore).
+// @ts-ignore
+const init = require('renovate/dist/workers/global/initialize');
+const origGlobalFinalize: any = init.globalFinalize;
+function patchedGlobalFinalize(config: any) {
+  globalRepos.forEach((repo, globalRepoURL) => {
+    logger.info(
+      { repo: repo.dir },
+      `deleting global defaults ${globalRepoURL}`
+    );
+    if (existsSync(repo.dir)) {
+      rmSync(repo.dir, { recursive: true });
+    }
+  });
+  origGlobalFinalize(config);
+}
+init.globalFinalize = patchedGlobalFinalize;
```

Comment:
Our patching of renovate is starting to get ...interesting :see_no_evil:

Reply:
yep, I couldn't find a direct way to add extra cleanup as part of the manager